tests/caasp/helm.pm

```diff
@@ -0,0 +1,55 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Use Helm, the Kubernetes Package Manager
+#   Using Helm to Install Applications
+#   Apps taken from: https://hub.kubeapps.com
+# Maintainer: Panagiotis Georgiadis <pgeorgiadis@suse.com>
+
+use parent 'caasp_controller';
+use caasp_controller;
+
+use strict;
+use utils;
+use testapi;
+
+sub run {
+    switch_to 'xterm';
+
+    record_info 'Test #1', 'Configure $HELM_HOME ~/.helm';
+    assert_script_run "helm version";
+    assert_script_run "helm init";
+
+    record_info 'Test #2', 'Verify that tiller is running on Kubernetes';
+    assert_script_run "kubectl get pods --all-namespaces | grep tiller";
```

Comment:
Testing manually there should be two pods running:

```bash
tux@ultron:~> kubectl get pods --all-namespaces | grep tiller
kube-system   tiller-797d6b5754-lxbql                                      1/1       Running   0          6m
kube-system   tiller-deploy-5b9d65c7f-sgrgf                                1/1       Running   0          2m
```

In that case I am thinking to create a *loop* waiting until both Pods are up and running:

```perl
for (1 .. 10) {
    last if script_run 'kubectl get pods --all-namespaces | grep tiller | grep -c Running';
    sleep 10;
}
```

Reply:
Yes, that should work. I use similar approach in kubernetes tests, where I am checking that ALL pods are running:
```perl
    for (1 .. 10) {
        last if script_run 'kubectl get pods | grep -q "0/\|1/2\|No resources"';
        sleep 10;
    }
    assert_script_run "kubectl get pods | tee /dev/tty | grep -c Running | grep $pods_count";
```